src/objects/core/zcl_abapgit_objects_files.clas.abap

```diff
@@ -497,4 +499,31 @@ CLASS zcl_abapgit_objects_files IMPLEMENTATION.
     ENDLOOP.
 
   ENDMETHOD.
+
+  METHOD get_i18n_properties_file.
+
+    DATA lv_lang TYPE laiso.
+    DATA lv_ext TYPE string.
+    FIELD-SYMBOLS <ls_file> LIKE LINE OF mt_files.
+
+    LOOP AT mt_files ASSIGNING <ls_file>.
+
+      " TODO: Maybe this should be in zcl_abapgit_filename_logic
```

Comment:
Does this need to be clarified, before it can be merged?

Reply:
copy&paste from method `read_i18n_files` above.

Reply:
i will take a look at this (https://github.com/abapGit/abapGit/issues/6865)